src/background/extensionController/extensionController.js

```diff
@@ -81,6 +93,23 @@ export class ExtensionController extends Component {
     return this.#mState.readOnly;
   }
 
+  get currentOnboardingPage() {
+    return this.#mCurrentOnboardingPage.readOnly; // true w/ or w/o readonly
+  }
+
+  nextOnboardingPage() {
+    this.#mCurrentOnboardingPage.set(this.#mCurrentOnboardingPage.value + 1);
+  }
+
+  finishOnboarding() {
+    this.#mCurrentOnboardingPage.set(4);
+    putIntoStorage(
+      4,
```

Comment:
Consider using constants to denote magic numbers Unless you have seen the figma it's not clear the max onboarding page is "4"

Reply:
Also what happens if we add a new onboarding page? Users would then be seeing that one no? 
(Not sure if this is wanted or not)

Reply:
Will update away from magic numbers, thanks.

We do want users to have the possibility of later seeing new onboarding pages - Lesley mentioned this was desired (and sometimes difficult) w/ MAC.